client/data/geo/use-geolocation-query.ts

```diff
@@ -13,8 +13,8 @@ export const useGeoLocationQuery = ( options = {} ) =>
 	useQuery< GeoLocationData >( {
 		queryKey: [ 'geo' ],
 		queryFn: () =>
-			globalThis
-				.fetch( 'https://public-api.wordpress.com/geo/' )
-				.then( ( response ) => response.json() ),
+			fetch( 'https://public-api.wordpress.com/geo/' ).then( ( response ) => response.json() ),
+		refetchInterval: Infinity,
+		meta: { persist: false },
```

Comment:
Is there a good reason not to persist the value? Doesn't having it immediately ready help, even if it's obsolete?

Reply:
> Is there a good reason not to persist the value? Doesn't having it immediately ready help, even if it's obsolete?

That's a good thought. I'd want to double-check the interaction between `refetchInterval` and `persist: true` to ensure that it would refresh after restoring from cache. Unless you happen to know off-hand.

Reply:
Yeah, any means of refetching (on focus, on mount, on reconnect, on in the background) will eventually overwrite the persisted data, so we might want to tinker with some of those. Note that we [might want to pass](https://tanstack.com/query/latest/docs/framework/react/plugins/persistQueryClient#how-it-works) `gcTime` instead of `refetchInterval` in that case, since that's what instructs how long to persist the data. Also the persistence client `maxAge` is 24 hours by default, it might make sense to double-check what that means - I think it meant that any older persisted data will be discarded.